test/specs/new/em_2char.md

```diff
@@ -20,10 +20,6 @@ _123 _
 
 _ 123_
 
-_1__
-
-*1**
```

Comment:
Are you sure you want to remove these tests?

It looks like @UziTech added these in PR #1181.

Reply:
They do not confirm to the commonmark spec, so trying to get them to pass was, obviously, not working.

Reply:
Maybe we can update the output of these two tests to match commonmark's output:

```html
<p><em>1</em>_</p>
<p><em>1</em>*</p>
```

Reply:
We could... but this PR does not parse it correctly either.

Since it wasn't correct in the earlier version, I feel it would be OK to let this one slide for now (this PR isn't breaking anything that was working before), and work on fixing that test in a future development. Getting this one obscure test to work that wasn't even working before involves a lot of obnoxious workarounds and I don't know if it's worth focusing on it for this PR.

Reply:
Yep that makes sense. I think this corner case will probably affect very few people so it can be addressed in a different PR 👍